### DISCUSSION TITLE
Higress architecture diagram and workflow

### DISCUSSION THREAD

**Discussion by bearslyricattack** [alibaba/higress#1638] [General]
The current developer guidelines for Higress are excellent in terms of source code reading, compilation, and execution. However, there is no system architecture diagram or workflow, and due to significant differences in design concepts, this part cannot be obtained by reading LSTIO or envoy. I hope to supplement this content in the documentation.

**Comment by johnlanni** [comment]
We previously organized some Higress-related tutorials; let's see if we can extract some content from the chapters to put on GitHub to help developers understand the architecture
cc @2456868764

**Comment by johnlanni** [comment]
You can check this document: https://github.com/alibaba/higress/blob/main/docs/architecture.md
